Dockerfile

```diff
@@ -143,6 +144,7 @@ RUN --mount=from=containerd-src,src=/usr/src/containerd,readwrite --mount=target
   && git checkout -q "$CONTAINERD_VERSION" \
   && make bin/containerd \
   && make bin/containerd-shim \
+  && make bin/containerd-shim-runc-v2 \
```

Comment:
what's this for? afaik containerd can't  do feature detection for shim versions?

Reply:
We don't need shim v2 atm, but I think we could have added shim v2 to the image when we updated containerd to v1.3.0, because eventually we will need it.

Reply:
> we could have added shim v2 to the image 

Added to what image. As these binaries should never go to the release images.

Reply:
reverted